src/bin/cargo/commands/rustc.rs

```diff
@@ -80,23 +80,15 @@ pub fn exec(gctx: &mut GlobalContext, args: &ArgMatches) -> CliResult {
         compile_opts.build_config.requested_profile = InternedString::new("dev");
     }
     let target_args = values(args, "args");
-    compile_opts.target_rustc_args = if target_args.is_empty() {
```

Comment:
It has to be wrapped with additional parentheses so doesn't seem more readable to me. The clippy lint [`if_then_some_else_none`](https://rust-lang.github.io/rust-clippy/master/index.html#/if_then_some_else_none) also only hints for the pattern of `if-some-else-none` not for the reversed.

Is there any other purpose of this change? Otherwise I tend to close this. Thank you.

Reply:
Agreed, I've found that if `!` is used, I would rather have an `if` / `else if`